src/main/java/io/vertx/ext/mail/impl/SMTPConnectionPool.java

```diff
@@ -55,7 +56,7 @@
     this.config = config;
     maxSockets = config.getMaxPoolSize();
     keepAlive = config.isKeepAlive();
-    prng = new PRNG(vertx);
```

Comment:
Perhaps you should keep this, and at the `close` of the pool, then also close the `PRNG` object to avoid re-seediing in background when not needed.

Reply:
Got it :)